src/core/config.ts

```diff
@@ -17,6 +17,17 @@ export interface ClientOptions {
   logLevel?: LogLevel;
 }
 
+/**
+ * Configuration options for the chat client, including internal options.
+ */
+interface ClientOptionsWithInternal extends ClientOptions {
+  /**
+   * Boolean flag to indicate whether the client has been created with the React chat provider.
+   * @default false
+   */
+  useReact?: boolean;
```

Comment:
Is this still used as it was meant to?

Reply:
No, now we update using an internal method, I'll remove this completely